disnake/types/widget.py

```diff
@@ -36,9 +35,15 @@ class WidgetChannel(TypedDict):
     position: int
 
 
+class WidgetActivity(TypedDict):
+    name: str
+
+
 class WidgetMember(User, total=False):
     nick: str
-    game: Activity
+    # `activity` is used starting api v8, `game` is used before
+    activity: WidgetActivity
+    game: WidgetActivity
```

Comment:
Widgets are a bit of a special case since they are sometimes used with the unversioned url, so I've opted to support both here in case a user passes their own data into `Widget.__init__` and doesn't use `fetch_widget`.
This is what the client does as well:
![image](https://user-images.githubusercontent.com/8530778/189457856-fe0ccd1e-77ff-4ddf-99db-12e6ae9a5afd.png)

Reply:
please add to the existing comment in the code that we intentionally have the typings for pre-v9.

Reply:
https://github.com/DisnakeDev/disnake/pull/736/commits/5bbb291e099f79026b21b09f2e06ac3617b4b9f9